docs/morph.md

```diff
@@ -1,18 +1,18 @@
-# N3 NeoFS side chain privnet service
+# N3 FrostFS side chain privnet service
 A single-node N3 privnet deployment, running on
-[neo-go](https://github.com/nspcc-dev/neo-go). Represents N3 NeoFS SideChain.
+[neo-go](https://github.com/nspcc-dev/neo-go). Represents N3 FrostFS SideChain.
 
 Contracts deployed:
-- Alphabet (AZ) [contract](https://github.com/nspcc-dev/neofs-contract/tree/master/alphabet)
-- Audit [contract](https://github.com/nspcc-dev/neofs-contract/tree/master/audit)
-- Balance [contract](https://github.com/nspcc-dev/neofs-contract/tree/master/balance)
-- Container [contract](https://github.com/nspcc-dev/neofs-contract/tree/master/container)
-- Netmap [contract](https://github.com/nspcc-dev/neofs-contract/tree/master/netmap)
-- NeoFSID [contract](https://github.com/nspcc-dev/neofs-contract/tree/master/neofsid)
-- Proxy [contract](https://github.com/nspcc-dev/neofs-contract/tree/master/proxy)
-- Reputation [contract](https://github.com/nspcc-dev/neofs-contract/tree/master/reputation)
+- Alphabet (AZ) [contract](https://github.com/TrueCloudLab/frostfs-contract/tree/master/alphabet)
+- Audit [contract](https://github.com/TrueCloudLab/frostfs-contract/tree/master/audit)
+- Balance [contract](https://github.com/TrueCloudLab/frostfs-contract/tree/master/balance)
+- Container [contract](https://github.com/TrueCloudLab/frostfs-contract/tree/master/container)
+- Netmap [contract](https://github.com/TrueCloudLab/frostfs-contract/tree/master/netmap)
+- FrostFSID [contract](https://github.com/TrueCloudLab/frostfs-contract/tree/master/neofsid)
```

Comment:
I think here we should keep the original name for now, not sure about rebranding of this conract.